Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs

```diff
@@ -80,6 +86,11 @@ public sealed partial class AdminVerbSystem
     [Dependency] private readonly SuperBonkSystem _superBonkSystem = default!;
     [Dependency] private readonly SlipperySystem _slipperySystem = default!;
 
+    private const string ActionViewLawsProtoId = "ActionViewLaws";
+    private const string CrewsimovLawset = "Crewsimov";
```

Comment:
```suggestion
    private const EntProtoId ActionViewLawsProtoId = "ActionViewLaws";
    private const ProtoId<SiliconLawsetPrototype> CrewsimovLawset = "Crewsimov";
```
Save those as prototypes, not strings